apps/total-typescript/src/trpc/routers/stackblitz-resource.ts

```diff
@@ -1,23 +0,0 @@
-import {publicProcedure, router} from '@skillrecordings/skill-lesson'
-import {z} from 'zod'
-import {getExercise} from '../../lib/exercises'
-
-export const stackblitzResourceRouter = router({
```

Comment:
I replaced this with `resourcesRouter` which we used on PT and implies more generic usage.